### ISSUE TITLE
Multiverse: First Support

### ISSUE THREAD

**Issue by zhoub** [pypeclub/OpenPype#2909]
**Is your feature request related to a problem? Please describe.**
Support Multiverse

**Describe the solution you'd like**
Support Multiverse

**Describe alternatives you've considered**
No alternative

**Additional context**
The PRs would come from https://github.com/j-cube/OpenPype

**Comment by zhoub** [comment]
https://github.com/pypeclub/OpenPype/pull/2908

**Comment by BigRoy** [comment]
With #2908 merged should we close this issue? Or are we waiting on additional implementation details?

**Comment by pberto** [comment]
No, let's keep this issue open as we are doing some improvements, next in line:

1. merge paolo's branch with the naming changes & docs -- 
2. add to the creator's set the option to chose the file format (USD / USDA for the override and composition and USD / USDZ for the asset one).
3. solve the issue of composition displaying a empty stage cause by either namespace or reference file path

Once 1 2 3 are done we wil make another PR.

**Comment by pberto** [comment]
Se should also try to understand why do we see (abc) near the Multiverse item. I assume this loader menu is not contact sensitive: I am trying to Load a Maya model I just wrote (.ma) -- I n any case our menu item should list usd and abc or only usd or nothing. Not just abc.
<img width="922" alt="image" src="https://user-images.githubusercontent.com/254052/166219503-dbb77713-a61b-4573-a8c4-59e830b33b84.png">

**Comment by BigRoy** [comment]
The representation name between the brackets (like `abc` and `ma`) is to decide on which file to act with the loader. The Multiverse loader only is linked to `abc` and `usd` in the [Loader](https://github.com/pypeclub/OpenPype/blob/develop/openpype/hosts/maya/plugins/load/load_multiverse_usd.py#L21) and thus would only show for those options.

The Model subset does not have an exported `usd` file by default since the USD extractor does not trigger for the model family but [only for the `usd` family](https://github.com/pypeclub/OpenPype/blob/develop/openpype/hosts/maya/plugins/publish/extract_multiverse_usd.py#L15). Thus when right clicking on a model that has no `usd` representation it will not show the _Read USD by Multiverse (usd)_ entry since it does not exist.

Does that answer your question?

**Comment by pberto** [comment]
Kind. My question now is why abc is displayed for the model subset?

**Comment by BigRoy** [comment]
The model family publishes both a .ma and .abc so that's why.

**Comment by pberto** [comment]
Understood.
